src/ui/sass/modules/_SearchBar.scss

```diff
@@ -213,13 +213,21 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
   &-LoadingIndicator-AnimatedIcon 
   {
     stroke-dasharray: 208;
+    /* IE11 specific styling */  
+    _:-ms-fullscreen, :root & { 
+      /**
+       * IE11 does not support stroke-* properties in animation, so use smaller stroke
+       * length to simulate a fixed gap instead of using stroke-dashoffset in animation.
+       */
+      stroke-dasharray: 150;
```

Comment:
Out of curiosity, could we remove the `stroke-dashoffset` from the animation SVG and simply use this styling regardless of browser? Would that be contrary to best practices? I ask because having browser-specific styling can get a bit tricky.

Reply:
it would simplify the loading indicator (i.e. the line that spin in a circle wouldn't shrink and expand) in comparison to what product wanted from the spec. I could hold off on this and ask rose when she's back about this change

Reply:
Ignore me.